### ISSUE TITLE
Custom Labels in Vertex AI Pipeline PipelineJobSchedule

### ISSUE THREAD

**Issue by rituraj17** [googleapis/python-aiplatform#2929]
Hi All, 
I would like to know the steps involved in adding custom labels to a Vertex AI pipeline’s PipelineJobSchedule. Can anyone please provide me with the necessary guidance as it's not working when I am adding inside the Pipelinejob parameters?

#### Environment details

  - OS type and version: Canonical, Ubuntu, 22.04 LTS, amd64 jammy image built on 2023-09-08
  - Python version: Python 3.10.12
  - pip version: pip 22.0.2 
  - `google-cloud-aiplatform` version: 1.36.1

#### Code example
```
# https://cloud.google.com/vertex-ai/docs/pipelines/schedule-pipeline-run#create-a-schedule
pipeline_job = aiplatform.PipelineJob(
  template_path="COMPILED_PIPELINE_PATH",
  pipeline_root="PIPELINE_ROOT_PATH",
  display_name="DISPLAY_NAME",
  labels={"name":"test_xx"}
)
pipeline_job_schedule = aiplatform.PipelineJobSchedule(
  pipeline_job=pipeline_job,
  display_name="SCHEDULE_NAME"
)

pipeline_job_schedule.create(
  cron="TZ=CRON",
  max_concurrent_run_count=MAX_CONCURRENT_RUN_COUNT,
  max_run_count=MAX_RUN_COUNT,
)
```
I am adding the labels during the PipelineJob process but when I am running the PipelineJobSchedule step the labels are either removed or overwritten with only <"vertex-ai-pipelines-run-billing-id"> label?

Also when I just submit the pipeline job by using the below script it works properly 
```
pipeline_job.submit()
```

Also I posted my question on[ Stack Overflow](https://stackoverflow.com/questions/77391156/custom-labels-in-vertex-ai-pipeline-pipelinejobschedule), but unfortunately, I did not receive a solution.

**Comment by KevinGrantLee** [comment]
Hi, thank you for bringing this to our attention - we are working on a fix.

**Comment by KevinGrantLee** [comment]
The fix was submitted and it should be rolled out in the next vertex sdk release. Feel free to close this issue when the next release rolls out.

**Comment by rituraj17** [comment]
@KevinGrantLee - Is the changes been released?

**Comment by KevinGrantLee** [comment]
Yes the fix has been released in the newest version 1.3.7.0.
https://github.com/googleapis/python-aiplatform/releases/tag/v1.37.0

Closing this issue now.
